File: src/FrontEnd/src/app/commands/dashboard/GetDashBoardResult.ts
import { GetEquipmentResumed } from "../../results/equipment/GetEquipmentResumed.model";

export class GetDashBoardResult {
    ListOfAndroid: GetEquipmentResumed[];
    ListOfManufacturers: GetEquipmentResumed[];
    ListOfOnlineOfflineDevices: GetEquipmentResumed[];
}

File: src/BackEnd/AppFullStackDemo.Domain/Results/DashBoard/GetDashBoardResult.cs
using System.Collections.Generic;

namespace AppFullStackDemo.Domain.Results.DashBoard
{
    public class GetDashBoardResult
    {
        public List<GetDashBoardInfo> ListOfAndroid { get; set; }
        public List<GetDashBoardInfo> ListOfManufacturers { get; set; }

        public GetDashBoardResult(List<GetDashBoardInfo> listOfAndroids, List<GetDashBoardInfo> listOfManufacturers)
        {
            ListOfAndroid = listOfAndroids;
            ListOfManufacturers = listOfManufacturers;
        }
    }
}